### PR TITLE
[ADD] surveys: created survey analysis doc for 17

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3835022&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

- This is a brand new, from-scratch doc to cover Survey Analysis in the Surveys application.

### CODE REVIEW

File: content/applications/marketing/surveys/analysis.rst [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+specific survey, along with a collection of pertinent information related to each one.
+
+.. image:: analysis/participations-page-singular-survey.png
+   :align: center
+   :alt: The Participations page for a singular survey in the Odoo Surveys application.
+
+Here, users can view information related to individual participants who took that specific survey,
+and if they desire to see a more detailed breakdown of their various answers and responses, they can
+click on any desired participant, and Odoo reveals a separate page showing that participant's survey
+details, along with their submitted answers.
```
Comment (jero-odoo) [inline content/applications/marketing/surveys/analysis.rst:233]:
```suggestion
Here, users can view information related to individual participants who took that specific survey.
If they desire to see a more detailed breakdown of their various answers and responses, they can
click on any participant, and Odoo reveals a separate page showing that participant's survey
details, along with their submitted answers.
```

File: content/applications/marketing/surveys/analysis.rst [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+To view a consolidated list of all participants of every survey in the database, navigate to
+:menuselection:`Surveys app --> Participations`. Here, every survey in the database is shown in a
+default nested list. Beside each survey title has the number of participants in parenthesis.
+
+.. image:: analysis/participations-page-all-surveys.png
+   :align: center
+   :alt: The Participations page for all surveys in the Odoo Surveys application.
+
+When a survey is un-nested from this list, by clicking the survey title, the corresponding
+participants, along with their response-related data for that survey appears on the page.
```
Comment (jero-odoo) [inline content/applications/marketing/surveys/analysis.rst:248]:
```suggestion
participants, along with their response-related data for that survey, appear on the page.
```

**Review by jero-odoo — CHANGES_REQUESTED** [review]
Hey @ksc-odoo great job on this doc! I had a few comments, a few are grammar/spelling corrections but a lot are suggestions so you can do with them what you choose! Let me know when you are ready for me to take another look :+1:

File: content/applications/marketing/surveys/analysis.rst [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+.. note::
+   Even if there is no configured *correct* response for question of this nature, the
+   :guilabel:`Correct` indicator still appears, although, it displays a `0`.
+
+   This would occur for opinion-based questions, like `When would be a good time to hold another
+   sale?`
+
+If there is only one correct response to a multiple choice question, those results and responses are
+represented by a :guilabel:`Pie Graph`. The correct answer is indicated by a :guilabel:`✔️
+(checkmark)` icon next to the right answer option, in the legend above the graph.
```
[the remark this answers is not in the document]
Comment (ksc-odoo) [reply content/applications/marketing/surveys/analysis.rst:143]: That's a GREAT point! Thanks for the suggestion!

**Comment by ksc-odoo** [comment]
Thanks for all the awesome suggestions, @jero-odoo -- I implemented all your edits, and think this is ready for another look from you, whenever you get a chance. Thanks again! 👍

**Comment by jero-odoo** [comment]
@ksc-odoo Looks good to me! ready for the next round :+1:

**Comment by ksc-odoo** [comment]
Thanks, @jero-odoo -- This one's ready for Tech Review @samueljlieber 👍

**Review by samueljlieber — APPROVED** [review]
Nice job @ksc-odoo, Im approving and pushing up a commit quickly resolving a merge conflict on `surveys.rst` from #8381. 👍
.....

**Pull request closed without merging**